### PR TITLE
Fix/exome genome options

### PR DESCRIPTION
**Pull request by iku000888**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ Here is a repl session that adds more context.
  :error-sum 0.023682681269721964}
 ```
 
+### Test data maintenance
+
+Assuming deconstructSigs is installed, run `Rscript R-reference/R_answers.R `.
+
```
Comment (federkasten) [inline README.md]: I have removed `R-reference` in 5875c62 to simplify clj-deconstruct-sigs codebase (and make programming language's ratio on GitHub "*Clojure 100%*" 😉  )

I think it's better to maintain these R scripts in another repository. What do you think?
If it’s ok with you, could you remove them?

File: src/clj_deconstruct_sigs/core.cljc [hunk 2]
```diff
@@ -83,6 +84,38 @@
                                (or (<= 0.01 t) (<= s 0.2)))))
             [sig-idx sig])))))
 
+(def ^:private ^:const trans-patterns
+  (for [[ref alts] [[\C "AGT"] [\T "ACG"]], alt alts, before "ACGT", after "ACGT"]
+    {:ref ref, :alt alt, :before before, :after after}))
+
+(defn normalize-data [tumor {:keys [count-method tri-counts]}]
```
Comment (federkasten) [inline src/clj_deconstruct_sigs/core.cljc]: `noramalize-data` name is generally a bit. I think should be `normalize-tri-acounts` or something other.

File: src/clj_deconstruct_sigs/data/tri_counts.cljc [hunk 3]
```diff
@@ -0,0 +1,78 @@
+(ns clj-deconstruct-sigs.data.tri-counts)
+
+;; Hardcoded exome and genome trinucleotide context counts based on hg19,
+;; bundled together with the original deconstructSigs
```
Comment (federkasten) [inline src/clj_deconstruct_sigs/data/tri_counts.cljc:4]: It's just a comment, tri-counts with hg38 is better. We are using hg38 on Chrovis. Could you submit an issue about it?

File: src/clj_deconstruct_sigs/core.cljc [hunk 4]
```diff
@@ -83,6 +84,38 @@
...
+  (let [normalize-fn (fn [data]
+                       (let [foo (double-array tumor)
```
Comment (federkasten) [inline src/clj_deconstruct_sigs/core.cljc]: `foo` ?

File: src/clj_deconstruct_sigs/core.cljc [hunk 5]
```diff
@@ -83,6 +84,38 @@
...
+                             wes (double-array (mapv
```
Comment (federkasten) [inline src/clj_deconstruct_sigs/core.cljc]: `weights` ?

**Review by federkasten — COMMENTED** [review]
Thanks. The codes looks fine and works well 👍  I leave some suggestions.

File: README.md (hunk 1 above)
Comment (iku000888) [reply README.md]: Sounds good! Will do.

**Review by federkasten — APPROVED** [review]
LGTM 🙆‍♂

**Pull request merged**